go/types.go

```diff
@@ -238,6 +241,7 @@ type LogEntry struct {
 	Leaf     MerkleTreeLeaf
 	X509Cert *x509.Certificate
 	Precert  *Precertificate
+	JSONData []byte
```

Comment:
This is the object that holds the serialized JSON, so it's not an `interface{}` at this point.